pkgs/racket-doc/file/scribblings/glob.scrbl

```diff
@@ -0,0 +1,53 @@
+#lang scribble/doc
+@require["common.rkt"]
+
+@title[#:tag "glob"]{Globbing}
+
+@defmodule[file/glob]{The @racketmodname[file/glob] library implements Unix-style
+wildcards for file or directory names within @racket[path-string] values.
+Supported wildcards are:
+@itemlist[
+@item{
+  @racket{*} matches any sequence of characters.
+}
+@item{
+  @racket{?} matches any single character.
+}
+@item{
+  Square bracket-delimited character groups, e.g. @racket{[abc]}, match any character
+  within the group. The square brackets have the same meaning in globs as in
+  regular expressions, see @secref["regexp-syntax" #:doc '(lib "scribblings/reference/reference.scrbl")].
+}
```

Comment:
You should also add {a,b}-{c,d}.rkt as a way to write a-c.rkt a-d.rkt b-c.rkt b-d.rkt